public/app/features/plugins/admin/components/VersionInstallButton.tsx

```diff
@@ -61,7 +61,7 @@ export const VersionInstallButton = ({
       schema_version: '1.0.0',
     };
 
-    if (!installedVersion) {
+    if (!installedVersion || gt(version.version, installedVersion)) {
       reportInteraction(PLUGINS_VERSION_PAGE_INSTALL_INTERACTION_EVENT_NAME, trackProps);
```

Comment:
nit: can we rename this to PLUGINS_VERSION_PAGE_UPGRADE_INTERACTION_EVENT_NAME ?